pod/perlfunc.pod

```diff
@@ -10177,6 +10183,21 @@ file (possibly after a C<package> statement or any amount of whitespace or
 comment), so that its effects happen first, and other pragmata are applied
 after it.
 
+Furthermore, once the version is 5.39.0 or higher, no later C<use VERSION>
+declaration within the same scope is permitted to lower the version again.
+This is because any imported functions from C<builtin> would need to be made
+invisible again.  If you wish to use a higher version only in a small
+section of a file, enclose that section in its own lexical scope.  Once
+outside the scope again afterwards, any imported effects are no longer
```

Comment:
perhaps "any lexically imported effects" or "any imported effects from use VERSION" so as not to be misleading

Reply:
Ahyes; reworded in 3fadae8c81 (will fixup before merge)